packages/components/MIGRATING.md

```diff
@@ -162,6 +162,108 @@ return (
 </Flyout.Root>
 ```
 
+### Dialog
```

Comment:
Please move the block to be in alphabetical order and update the table of content

Reply:
changed